skt/kernelbuilder.py

```diff
@@ -124,6 +129,15 @@ def make_tinyconfig(self):
         logging.info("building tinyconfig: %s", args)
         subprocess.check_call(args)
 
+    def get_build_arch(self):
+        """Determine the build architecture for the kernel build."""
+        # Detect an architecture specified for the kernel build (for cross
+        # compiling kernels) and set the build arch
```

Comment:
The function itself isn't setting anything, so this part of the comment is confusing. Can you mention cross-compilation in the docstring and remove the additional multiline comment instead?

Reply:
Sure.